### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by XDIJK**

### CODE REVIEW

File: 4517_Sprint/04 - xander/OCP-CH4/src/com/company/Demo.java [hunk 1]
```diff
@@ -0,0 +1,116 @@
+package com.company;
+
+import java.util.*;
```
Comment (ssoegriemsingh) [inline 4517_Sprint/04 - xander/OCP-CH4/src/com/company/Demo.java]: Should watch out with this, can lead to mistakes.

File: 4517_Sprint/04 - xander/OCP-CH4/src/com/company/Demo.java [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+      }
+
+      return vinylArray;
+    }
+
+    public static String getRandomArtist() {
+      String[] artists = {"Michael Jackson","AC/DC","Pink Floyd","Whitney Houston","Meat Loaf","Eagles","Fleetwood Mac"
+          ,"Shania Twain","Led zeppelin", "Celine Dion", "The Beatles", "Adele", "Metallica", "Nirvana"
+          ,"Guns N' Roses", "ABBA", "Linkin Park", "Bob Marley", "Queen", "Tommpabeats"};
+
```
Comment (ssoegriemsingh) [inline 4517_Sprint/04 - xander/OCP-CH4/src/com/company/Demo.java]: Add extra tabs to have the array items below each other, for better readability.

File: 4517_Sprint/04 - xander/OCP-CH4/src/com/company/Demo.java [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+    Random r = new Random();
+
+    return albumNames[r.nextInt(albumNames.length)];
+  }
+
+  public static List<String> generateListOfRandomGenres() {
+  String[] genres = {"jazz","funk", "pop", "low-fi", "metal", "rap", "hip-hop", "vaporwave", "trap", "kpop", "rock"
+                      ,"death-metal", "dubstep", "ska", "soul", "blues"};
+
```
Comment (ssoegriemsingh) [inline 4517_Sprint/04 - xander/OCP-CH4/src/com/company/Demo.java]: Keep your coding style consistent. Either multiple tabs to have the items below one another or just one tab to indicate a new row. I'd go for the first option.

**Comment by ohsnapitskenny** [comment]
https://i.imgur.com/6fddGmo.png

**Pull request merged**